package.json

```diff
@@ -76,11 +76,7 @@
         "testcontainers": "^9.12.0",
         "tsx": "^4.6.2",
         "typescript": "^5.3.3",
-        "vitest": "^0.33.0"
-    },
-    "dependencies": {
-        "@babel/parser": "^7.22.5",
-        "@babel/traverse": "^7.22.5"
```

Comment:
I think they were here for the CLI at the very beggining, probably a leftover but I prefer to have a confirmation

Reply:
Confirmed, left over from before.